tools/run_server_container.sh

```diff
@@ -26,7 +26,9 @@ if [ "${QDRANT_VERSION}" == "dev" ]; then
     ssh -t "${SERVER_USERNAME}@${IP_OF_THE_SERVER}" "cd ./projects/vector-db-benchmark/engine/servers/${CONTAINER_NAME} ; $DOCKER_COMPOSE"
 else
     # else run natively in the server
-    DOCKER_QDRANT_STOP="docker stop qdrant-continuous || true"
-    QDRANT_BUILD="git checkout ${QDRANT_VERSION}; nohup mold -run cargo run --bin qdrant --release &"
-    ssh -t "${SERVER_USERNAME}@${IP_OF_THE_SERVER}" "cd ./projects/qdrant; ${DOCKER_QDRANT_STOP}; $QDRANT_BUILD"
+    QDRANT_STOP="docker stop qdrant-continuous; pkill qdrant"
+    QDRANT_BUILD="git checkout ${QDRANT_VERSION}; source /root/.cargo/env; mold -run cargo build --bin qdrant --release"
```

Comment:
I think it might be better to start building from fresh copy, for the following reasons:

- solves updates of the repo
- do not require pre-existing configuration of the server
- potentially less error-prone, as cargo requires clearing sometimes

Ideally, it might worth considering building docker from source, as in this case we can also drop the requirement of having pre-configured mold and cargo, so it should be easily extendable to a case when we spawn new machines for a benchmark on-spot